firefox-ios/Client/Coordinators/ShareExtensionCoordinator.swift

```diff
@@ -98,7 +98,7 @@ class ShareExtensionCoordinator: BaseCoordinator,
         let themeColors = themeManager.currentTheme(for: windowUUID).colors
         let colors = SendToDeviceHelper.Colors(defaultBackground: themeColors.layer1,
                                                textColor: themeColors.textPrimary,
-                                               iconColor: themeColors.iconPrimary)
+                                               iconColor: themeColors.iconDisabled)
```

Comment:
just checking - using the disabled color here was intentional yes?

Reply:
Yes, it's intentional. It was discussed with @cwzilla